CSharp/VirtoCommerce.Module.Template/src/Web/module.manifest

```diff
@@ -2,32 +2,26 @@
 <module>
     <id>$ext_safeprojectname$</id>
     <version>1.0.0</version>
-    <platformVersion>2.13.56</platformVersion>
+    <version-tag></version-tag>
+    <platformVersion>3.0.0</platformVersion>
     <dependencies>
-      <dependency id="VirtoCommerce.Core" version="2.25.32" />
-      <dependency id="VirtoCommerce.Pricing" version="2.18.10" />
+<!--        <dependency id="VirtoCommerce.Core" version="3.0.0" /> -->
     </dependencies>
 
     <title>$ext_projectname$</title>
     <description>Enter your module description here.</description>
     <authors>
-      <author>$username$</author>
+        <author>$username$</author>
     </authors>
     <owners>
       <owner>$registeredorganization$</owner>
     </owners>
-  
+
     <releaseNotes>First version.</releaseNotes>
     <copyright>Copyright © $year$ $registeredorganization$. All rights reserved</copyright>
     <tags>extension module</tags>
```

Comment:
these are missing in powershell template. Need to synch with it: add or remove.

Reply:
Fixed